app/src/main/java/com/epfl/drawyourpath/challenge/Trophy.kt

```diff
@@ -0,0 +1,19 @@
+package com.epfl.drawyourpath.challenge
+
+/**
+ * class representing trophies that can be earned by User
+ */
+enum class Trophy(
+    val trophyName: String,
+    val description: String,
+    val imagePath: String = "trophy/trophy.png"
+) : java.io.Serializable {
+
+    THEFIRSTKM("The First Kilometer", "run a total of 1 kilometer"),
+    TENKM("10 Kilometers", "run a total of 10 kilometers"),
+    MARATHON("MARATHON", "run a total of 42.195 kilometer"),
+    THEFIRSTHOUR("The first Hour", "run for a total of 1 hour"),
+    THEFIRSTPATH("The first Path", "draw your fisrt path")

```

Comment:
Maybe you can add a trophy for the Ralph marathon and the 5km who are very popular distance which are present in the most of the running competition.

Reply:
For now, those are placeholder, it will eventually be more populated